qa/standalone/scrub/osd-recovery-scrub.sh

```diff
@@ -42,11 +142,12 @@ function TEST_recovery_scrub() {
     OBJECTS=4
 
     setup $dir || return 1
-    run_mon $dir a --osd_pool_default_size=1 --mon_allow_pool_size_one=true || return 1
+    run_mon $dir a --osd_pool_default_size=1 --mon_allow_pool_size_one=true \
```

Comment:
does this test still need 32 PGs?

Reply:
I tried with 1 PG and the test failed.  Not sure why, but this works.

Reply:
If we can dig into the reason, we can make a follow-on fix for this. This is fine for now.